warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java

```diff
@@ -191,8 +191,12 @@ private static void verify(String name, int numArgs) {
                 break;
             case QueryFunctions.UNIQUE_FUNCTION:
             case QueryFunctions.UNIQUE_BY_DAY_FUNCTION:
-            case QueryFunctions.UNIQUE_BY_HOUR_FUNCTION:
```

Comment:
was hour intentionally removed?

Reply:
Nope, looks like I flubbed that. Will add it back in on next commit